manticore/core/cpu/x86.py

```diff
@@ -5744,6 +5675,7 @@ def __init__(self, memory, *args, **kwargs):
         :param machine:  machine code name. Supported machines: C{'i386'} and C{'amd64'}.
         '''
         super(AMD64Cpu, self).__init__(AMD64RegFile(aliases={'PC' : 'RIP', 'STACK': 'RSP', 'FRAME': 'RBP'},  ), memory, *args, **kwargs)
+        self._abi = AMD64ABI(self)
```

Comment:
There are multiple ABIs for x64; in order to support others than SystemV, I feel like this needs to be a ctor parameter, or not be a Cpu attribute at all.